### PR TITLE
edit contact request status

### PR DESCRIPTION
**Pull request by liela9**

### CODE REVIEW

File: server/routes/api/moderator/moderatorHandler.ts [hunk 1]
```diff
@@ -8,11 +15,32 @@ export default class Handler {
     this.dbHandler = new DBHandler();
   }
 
+  requestValidation = async (requestId: number, newStatus: string): Promise<Validation> => {
+    const contactRequest = await this.dbHandler.getContactRequest(requestId);
+    if (!contactRequest.length) {
+      return { isValid: false, message: "Incorrect id" };
+    }
+
+    if (!newStatus) {
```
Comment (shellytesler) [inline server/routes/api/moderator/moderatorHandler.ts:24]: First check if newStatus is undefined and then check the request id cause there you query the db

File: server/routes/api/moderator/moderatorHandler.ts [hunk 2]
```diff
@@ -1,5 +1,12 @@
 import DBHandler from "./moderatorDBHandler";
-import { User, ContactRequest } from "./moderatorTypes";
+import { User, ContactRequest, Validation } from "./moderatorTypes";
+
+enum RequestStatuses {
```
Comment (shellytesler) [inline server/routes/api/moderator/moderatorHandler.ts:4]: why did you create an enum and not an array?

**Review by shellytesler — CHANGES_REQUESTED** [review]

File: server/routes/api/moderator/moderatorHandler.ts (hunk 2 above)
Comment (liela9) [reply server/routes/api/moderator/moderatorHandler.ts:4]: because the elements are constants

**Review by shellytesler — APPROVED** [review]
Nice job!

**Pull request merged**